2018-09-28 11:11:23.667 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 11:11:23.667 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 11:11:56.121 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 11:11:56.121 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 10:08:28.339 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:08:38.353 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:08:48.363 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:08:58.372 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:09:08.377 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:09:09.917 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:09:18.381 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:09:28.388 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:09:38.393 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:09:48.396 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 10:09:58.407 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 11:10:49.537 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 11:12:06.077 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:24:47.696 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:24:47.699 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:25:47.920 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:25:47.923 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:31.184 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:31.184 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:31.214 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:31.214 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:31.217 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:31.217 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:31.217 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:31.217 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:31.248 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:31.248 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:43.645 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:43.645 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:43.645 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:43.645 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:43.660 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:43.660 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:43.660 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:43.660 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 13:54:43.676 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 13:54:43.676 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 16:59:34.846 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 16:59:44.868 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 16:59:44.872 | [main] | ERROR | dao.UserDao | DataBaseAcessError
2018-09-28 16:59:54.889 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 16:59:54.890 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 17:03:51.371 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:03:51.371 | [main] | ERROR | dao.UserDao | DatabaseConnectError
2018-09-28 17:04:01.388 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:04:01.390 | [main] | ERROR | dao.UserDao | DataBaseAcessError
2018-09-28 17:04:11.403 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:04:11.404 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 17:08:24.828 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:08:24.828 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 17:08:34.849 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:08:34.849 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 17:08:44.865 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:08:44.865 | [main] | ERROR | dao.UserDao | DatabaseConnectError
2018-09-28 17:08:54.880 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:08:54.880 | [main] | ERROR | dao.UserDao | DataBaseAcessError
2018-09-28 17:09:04.887 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:09:04.887 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 17:10:51.044 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:10:51.044 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 17:11:01.070 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:11:01.072 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
2018-09-28 17:11:11.089 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:11:11.089 | [main] | ERROR | dao.UserDao | DatabaseConnectError
2018-09-28 17:11:21.094 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:11:21.095 | [main] | ERROR | dao.UserDao | DataBaseAcessError
2018-09-28 17:11:31.101 | [main] | ERROR | dao.DBHelper | connectDb() SQLException
2018-09-28 17:11:31.103 | [main] | ERROR | dao.ResourceDao | DatabaseConnectError
